src/ScheduledTask/UpdatePaymentStatus/UpdatePaymentStatusTask.php

```diff
@@ -13,7 +13,7 @@ public static function getTaskName(): string
 
     public static function getDefaultInterval(): int
     {
-        if ('dev' === getenv('APP_ENV')) {
+        if ( getenv('APP_ENV') === 'dev' ) {
```

Comment:
Are blank spaces needed at the beginning and end of the `if` statements ?

Reply:
No, it's just maybe some formatting  issue